### PR TITLE
Moved the AIDA chatbot feature description (PL-39613) to the early ac…

### PR DESCRIPTION
**Pull request by deepakbolangady**
…cess section after adding the FF statement to it.

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/platform.md [hunk 1]
```diff
@@ -110,7 +108,9 @@ This item requires Harness Delegate version 79904. For information about feature
 
 ##### Early access
 
-This release does not include any early access features.
+- An AI-powered chatbot named Ask AIDA has been integrated to make searches within the Harness Docs. You can access the chatbot by clicking the icon located at the bottom-right corner of the screen. (PL-39613)
```
Comment (aimurphy) [inline release-notes/platform.md:111]:
```suggestion
- You can use **Ask AIDA**, an AI-powered chatbot, for help with configuration, troubleshooting, or other questions you have while using Harness. Ask AIDA intelligently searches the Harness Developer Hub for relevant documentation. You can access the chatbot through the AIDA icon in the bottom-right corner of the screen. (PL-39613)
```

File: release-notes/early-access.md [hunk 2]
```diff
@@ -30,6 +30,12 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
 
    This functionality is behind a feature flag, `DELEGATE_TASK_CAPACITY_CHECK`. When the feature flag is enabled, the task is broadcast every minute in Harness Manager until it expires.
 
+### Harness Platform, version 79916
+
+- An AI-powered chatbot named Ask AIDA has been integrated to make searches within the Harness Docs. You can access the chatbot by clicking the icon located at the bottom-right corner of the screen. (PL-39613)
```
Comment (aimurphy) [inline release-notes/early-access.md:35]:
```suggestion
- You can use **Ask AIDA**, an AI-powered chatbot, for help with configuration, troubleshooting, or other questions you have while using Harness. Ask AIDA intelligently searches the Harness Developer Hub for relevant documentation. You can access the chatbot through the AIDA icon in the bottom-right corner of the screen. (PL-39613)
```

**Review by SudheendraKatte — DISMISSED** [review]
Approved with a minor suggestion.

**Review by aimurphy — DISMISSED** [review]
Approved but the description was a bit flat. I thought it could be more exciting.

**Review by StrawberryPad — APPROVED** [review]

File: release-notes/early-access.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (deepakbolangady) [reply release-notes/early-access.md]: Very good edits, Sudhi! Thank you. I'll incorporate when we launch the chatbot. For now, though, we'll have to wait.

File: release-notes/platform.md (hunk 1 above)
Comment (deepakbolangady) [reply release-notes/platform.md]: Thank you, @aimurphy! I'll incorporate your edits when we roll this out.

**Pull request merged**